### DISCUSSION TITLE
Call requireModule to register build module

### DISCUSSION THREAD

**Discussion by daniluk4000** [nuxt/nuxt#19098] [Questions (Nuxt 2)]
### Is your feature request related to a problem? Please describe.

I want to call requireModule (or addModule) to register a buildModule, but don't understand, how to do that (don't see any documentation about that)

### Describe the solution you'd like

Allow to call requireModule to register build-only module (or improve docs about how to do that)

### Describe alternatives you've considered

I don't see any alternatives

**Comment by daniluk4000** [comment]
@pi0 sorry for pinging (again), I can't find how to add a build module in documentation. Still not possible?

**Comment by Piscinelove** [comment]
@daniluk4000 did you find a solution i have the same issue

**Comment by daniluk4000** [comment]
@Piscinelove No success sadly.
